### ISSUE TITLE
Setup script is confusing 

### ISSUE THREAD

**Issue by xavetar** [wukko/cobalt#38]
**Current and Crowdin (3.5.2). Test on YouTube and VK (audio, video):**

<img width="1160" alt="cobalt" src="https://user-images.githubusercontent.com/52722847/190920343-e1ebbbd7-c8da-43ba-8612-78a3d5fd1573.png">

**Trying to download from YouTube resulted in a warning:**

<img width="476" alt="cobalt" src="https://user-images.githubusercontent.com/52722847/190920615-1ae072ee-9c6f-49ac-8a24-9436606a2bbe.png">

After trying to update: "**npm install npm@latest -g**". There is no result, no matter what the order. Problem with code or with npm 8.19.2/node 18.9.0, maybe this version does not work correctly with this code.

Your service on [co.wukko.me](https://co.wukko.me/) is now work. And yes turn that **shit** off: https://github.com/wukko/cobalt/blob/current/src/modules/sub/currentCommit.js

**Comment by wukko** [comment]
in order to get youtube module to work, you have to do [this](https://twitter.com/justusecobalt/status/1565355332423274497).  i forgot to add it to readme. maybe i should've added it to setup script because it's basically required now.

vk should work without any modifications, are you sure it works on [co.wukko.me](https://co.wukko.me), but not when hosted locally?

and finally, what's wrong with commit info module? all data passed through to it is local and is never accessed or modified by the service user.

**Comment by wukko** [comment]
after looking at your screenshots again, you haven't followed all the requirements and setup steps. do that, then come back. 

both vk and youtube may not work because of missing stream hash in .env file that's genereated by the setup script and is required for streamable stuff to work.

**Comment by xavetar** [comment]
I have fixed inaccuracies in the setup script, according to which the setup becomes more understandable. The problem has been solved. Except that the user must still connect to the host, which was specified during the initial setup. So if the user defined the initial address as 127.0.0.1 and connected from a browser to localhost:9000, or from another machine in the internal/local network, the initiator will bump to the defined address 127.0.0.1, not counting the port. And originally the default connection type was specified even for localhost only by https, which caused connection problems and because of this. However, created a [pull request](https://github.com/wukko/cobalt/pull/39).
